taxi/forms.py

```diff
@@ -0,0 +1,52 @@
+import re
+
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+from django.forms import ModelForm
+
+from taxi.models import Car, Driver
+
+
+class DriverLicenseUpdateForm(ModelForm):
+    LICENSE_REQUIRED_LENGTH = 8
+
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        pattern = r"^[A-Z]{3}[0-9]{5}"
```

Comment:
small hint - never use re, if you can, since it's slowing down your python code.